statvar_imports/pennsylvania/pennsylvania_education/public_school_enrollment_by_county_grade_and_race_pvmap.csv

```diff
@@ -0,0 +1,27 @@
+key,p1,v1,p2,v2,,,,
+County,observationAbout,{Data},populationType,Student,measuredProperty,count,observationDate,2017
+Pre-Kindergarten Half Day,schoolGradeLevel,PreKindergarten,schoolDayType,HalfDay,value,{Number},,
```

Comment:
Can we use the existing schoolType with values FullDaySchool and HalfDaySchool values.